### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by Brennan1994** [HydrologicEngineeringCenter/HEC-FDA#21] [Show and tell]
My name is Brennan and I'm interested in testing GitHub Discussions.

**Comment by rnugent3** [comment]
My name is Richard, I'm a May Gemini, brunch is my favorite time of the day, and I am also interested in testing GitHub Discussions.

**Comment by HenryGeorgist** [comment]
My name is Will. I mostly create issue reports and review pull requests. I like working on community projects. My vision statement is "Collaboratively finding creative solutions to complex problems"

**Comment by codymccoy** [comment]
My name is Cody.
